lib/huginn_deposco_inventory_agent/deposco_inventory_agent.rb

```diff
@@ -194,7 +194,7 @@ def fetch_product_inventory(deposco_client, skus, business_unit)
             errors = []
 
             skus.each do |sku|
-                path = "/items/#{business_unit}/#{sku}/atps"
+                path = "/ctrl/getRestfulATP?businessUnit=#{business_unit}&itemNumber=#{sku}"
```

Comment:
@Jhalter5Stones We should probably just have the client build this in the url and pass the `business_unit` and `sku` as function parameters.